pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -88,7 +92,7 @@ let
 
     hardeningDisable = [ "all" ];
 
-    bazelFlags = [ "--config=opt" ]
+    bazelFlags = [ "--config=opt" "--config=monolithic" ]
```

Comment:
Please leave a comment on why is this necessary; let's make it optional for Darwin to avoid static dependencies as much as possible.